page/product.ejs

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <%- include('partials/head', {title: 'Product' }); %>
+    <%- include('partials/head', {title: 'Product', meta: content.meta_head }); %>
     <body>
```

Comment:
i think you forgot about faq page

Reply:
No, I didn't 
https://github.com/Neufund/commit.neufund.org/pull/53/files#diff-8d814f366939e25c10181bd637bc829dR3